policyengine_us/variables/gov/states/hi/tax/income/deductions/hi_deductions.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class hi_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii deductions"
+    unit = USD
+    documentation = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=15"  # Itemized Deduction
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=20"  # Standard Deduction
+    )
+    definition_period = YEAR
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.deductions.itemized
+
+        standard_deduction = tax_unit("hi_standard_deduction", period)
+        itemized_deduction = tax_unit("hi_itemized_deduction", period)
+        hi_agi = tax_unit("hi_agi", period)
```

Comment:
this might be earned income as opposed to agi

Reply:
use tax_unit_earned_income = tax_unit("tax_unit_earned_income", period) now